cvmfs/util/posix.h

```diff
@@ -148,7 +148,7 @@ std::vector<LsofEntry> Lsof(const std::string &path);
 bool ProcessExists(pid_t pid);
 void BlockSignal(int signum);
 void WaitForSignal(int signum);
-int WaitForChild(pid_t pid);
+int WaitForChild(pid_t pid, std::vector<int> sig_ok = std::vector<int>());
```

Comment:
```suggestion
int WaitForChild(pid_t pid, const std::vector<int> &sig_ok = std::vector<int>());
```

Reply:
I also saw this one, but I thought it wouldn't make much of a difference here.